### PR TITLE
Adding option to configure proxy

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1598

@dariomws Can you please review? Thanks

### CODE REVIEW

**Review by dariomws — APPROVED** [review]

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md [hunk 1]
```diff
@@ -68,7 +68,21 @@ If your network environment uses outbound proxy servers, additional configuratio
...
 #### Agent
 
-The agent supports outbound unauthenticated proxy servers but requires additional configuration after installation. For more information, see [Work with existing on-premises proxy servers](https://docs.microsoft.com/azure/active-directory/manage-apps/application-proxy-configure-connectors-with-proxy-servers).
+The agent supports outbound unauthenticated proxy servers but requires additional configuration after installation by running the ConfigureOutBoundProxy.ps1 script located in \Program Files\Microsoft Hybrid Service\ on the computer where the agent is installed, for example:
+
+```PowerShell
+PS C:\Program Files\Microsoft Hybrid Service\>.\ConfigureOutBoundProxy.ps1 -ProxyAddress http://proxyserver:8080
+```
+
+By running the script the following section will be added to Microsoft.Online.EME.Hybrid.Agent.Service.exe.config file located on the same folder:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md:77]:
```suggestion
By running the script, the following section will be added to the Microsoft.Online.EME.Hybrid.Agent.Service.exe.config file located in the same folder:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**